util/include/crypto/eddsa/EdDSA.hpp

```diff
@@ -25,4 +27,60 @@ class EdDSAPrivateKey : public SerializableByteArray<EdDSAPrivateKeyByteSize> {
 class EdDSAPublicKey : public SerializableByteArray<EdDSAPublicKeyByteSize> {
  public:
   EdDSAPublicKey(const EdDSAPublicKey::ByteArray& arr) : SerializableByteArray<EdDSAPublicKeyByteSize>(arr) {}
-};
\ No newline at end of file
+};
+
+/**
+ * @brief Generate hex format key from pem file.
+ *
+ * @tparam ByteArrayKeyClass
+ * @param pemKey
+ * @param KeyLength
+ * @return std::vector<uint8_t> Generated key.
+ */
+template <typename ByteArrayKeyClass>
+static std::vector<uint8_t> extractHexKeyFromPem(const std::string_view pemKey, size_t KeyLength) {
```

Comment:
the method name should reflect that it's for eddsa keys.

Reply:
He can also put it in a namespace (has the same effect)

Reply:
Changed in https://github.com/arc-vmware/concord-bft/pull/12/commits/7b00c7fa490e8abfda7a7da61799a0b6625826df